hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/RealtimeCompactedRecordReader.java

```diff
@@ -188,6 +189,7 @@ public long getPos() throws IOException {
 
   @Override
   public void close() throws IOException {
+    ((ExternalSpillableMap) deltaRecordMap).close();
```

Comment:
possible to add an ut for this?

Reply:
UT added. 
now we have no way to get the path of DiskMap in RealtimeCompactedRecordReader.
 so in the test we search the TMP file and check whether it has been deleted